### PR TITLE
Use the `tmt-lint` pre-commit hook

### PR DESCRIPTION
**Pull request by lukaszachy**

### CODE REVIEW

File: .pre-commit-config.yaml [hunk 1]
```diff
@@ -55,3 +55,8 @@ repos:
     hooks:
       - id: ruff
         args: [--fix]
+
+  - repo: https://github.com/teemtee/tmt.git
+    rev: 1.23.0
```
Comment (happz) [inline .pre-commit-config.yaml:60]: For 3rd party checks, I'm all for freezing their versions. When it comes to tmt and this kind of dogfooding, is there a risk of using `main`?

Comment (lukaszachy) [reply .pre-commit-config.yaml:60]: Would be nice, right? But: 
> [WARNING] The 'rev' field of repo 'https://github.com/teemtee/tmt.git' appears to be a mutable reference (moving tag / branch).  Mutable references are never updated after first install and are not supported.  See https://pre-commit.com/#using-the-latest-version-for-a-repository for more details.  Hint: `pre-commit autoupdate` often fixes this.

Comment (happz) [reply .pre-commit-config.yaml:60]: Aaaah... Ok, thanks. Nevermind then.

**Review by happz — APPROVED** [review]

**Review by psss — APPROVED** [review]
Nice, thanks!

**Pull request merged**